<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CredentialsEditPage</class>
 <widget class="QWizardPage" name="CredentialsEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string>Enter Credentials</string>
  </property>
  <property name="subTitle">
   <string>Enter the credentials to be sent to the target.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DropOnly</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::CopyAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>349</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="addButton">
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonFollowStyle</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_Add_Entry">
   <property name="icon">
    <iconset theme="list-add" resource="../../../build-KeepassTransfer-Desktop_Qt_5_11_2_GCC_64bit-Debug/clients/quick/clientsquick_qmlcache.qrc">
     <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../build-KeepassTransfer-Desktop_Qt_5_11_2_GCC_64bit-Debug/clients/quick/clientsquick_qmlcache.qrc"/>
 </resources>
 <connections/>
</ui>
